opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/AeSeHCouplerModule.java

```diff
@@ -0,0 +1,79 @@
+package org.opt4j.optimizers.ea.aeseh;
+
+import org.opt4j.core.config.annotations.Info;
+import org.opt4j.core.config.annotations.Name;
+import org.opt4j.core.start.Constant;
+import org.opt4j.optimizers.ea.Coupler;
+import org.opt4j.optimizers.ea.CouplerModule;
+
+/**
+ * Binds the {@link AeSeHCoupler} as the {@link Coupler}
+ * 
+ * @author Fedor Smirnov
+ *
+ */
+@Name("Epsilon Neighborhood")
+public class AeSeHCouplerModule extends CouplerModule {
```

Comment:
I would propose to also rename this coupler to EpsilonNeighborhoodCouplerModule and EpsilonNeighborhoodCoupler, resp.